src/utils/logging.ts

```diff
@@ -111,7 +118,7 @@ const logs = new Map<string, Log>();
 // dynamically create a new log as necessary.  All property accesses on the
 // Proxy get shunted to the `get()` method, which can handle it similar to
 // Ruby's method_missing.
-export default new Proxy<Module>({}, {
+export const Logging = new Proxy<Module>({}, {
```

Comment:
Given that _most_ importers only need the main `Logging`, wouldn't `export default` make sense here? `background.ts` can do `import Logging, { setLogLevel } from …`

See `src/window/index.ts` for an example: `import Electron, { BrowserWindow, app, shell } from 'electron';`

Reply:
Ah I didn't know you could do that. I'll implement this.